org.signal.Signal.json

```diff
@@ -0,0 +1,71 @@
+{
+    "app-id": "org.signal.Signal",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "signal",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications"
```

Comment:
Does it support tray icons? If so you'll need the env var and talk names from Discord for example.

Reply:
It doesn't, at least in this version. (There is pull request that adds it but not merged yet.)